.github/workflows/package.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         go-version: [1.16.x]
-        platform: [macos-10.15, ubuntu-16.04, windows-2019]
+        platform: [macos-10.15, ubuntu-18.04, windows-2019]
```

Comment:
Is there a reason you updated to 18? This will mean Wombat will be unable to run on older versions of linux.

Reply:
> Is there a reason you updated to 18? This will mean Wombat will be unable to run on older versions of linux.

github-action not supported ubuntu16
detail see https://github.com/actions/virtual-environments

Reply:
Damn. Thanks for the details.

Reply:
- [x] modify ChangeLog